tests/patcher_test.py

```diff
@@ -536,3 +536,7 @@ def test_patcher_existing_locks():
 
 def test_patcher_existing_locks_exception():
     tests.run_isolated("patcher_existing_locks_exception.py")
+
+
+def test_patcher_threading_subclass_done():
```

Comment:
This is failing in CI.

Reply:
Tsk. That's because dead greenlets evaluate falseish.

I had the code structured differently earlier, when I was testing.